modules/lang-painless/src/main/java/org/elasticsearch/painless/phase/DefaultUserTreeToIRTreePhase.java

```diff
@@ -862,7 +835,7 @@ public void visitAssignment(EAssignment userAssignmentNode, ScriptScope scriptSc
                 stringConcatenationNode.setLocation(irStoreNode.getLocation());
                 stringConcatenationNode.setExpressionType(String.class);
                 irCompoundNode = stringConcatenationNode;
-            // handles when the operation is mathematical
+                // handles when the operation is mathematical
```

Comment:
maybe put these after the control structure on the same line so intellij doesn't keep "intelligently" moving them.

Reply:
Fixed.